doc_utils/doc_utils.py

```diff
@@ -2,6 +2,8 @@
 
 from typing import Dict, List, Any
 
+from math import isnan
```

Comment:
is pd.isna. `pd.isna` will account for all NaNs including Pandas nan, numpy nan, math nan